packages/loot-design/src/components/budget/rollover/BudgetSummary.js

```diff
@@ -113,7 +113,7 @@ function TotalsList({ prevMonthName, collapsed }) {
           binding={rolloverBudget.forNextMonth}
           formatter={value => {
             let n = parseInt(value);
-            n = isNaN(n) ? 0 : -n;
+            n = isNaN(n) || n === 0 ? 0 : -n;
             let v = format(n, 'financial');
 
             return n > 0 ? '+' + v : n === 0 ? '-' + v : v;
```

Comment:
I'd change this function to be something along the lines of the below which feels more readable to me.

```
let formatter = (value) => {
  let n = parseInt(value) || 0;
  let v = format(Math.abs(n), 'financial');
  return n >= 0 ? '-' + v : '+' + v;
}
```

Reply:
Neat, and `NaN` is indeed falsy.